src/Dispatcher/Masa.Contrib.Dispatcher.Events/Internal/InitializeServiceProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Dispatcher.Events.Internal;
+
+internal class InitializeServiceProvider : IInitializeServiceProvider
+{
+    private bool _state;
```

Comment:
What's the state mean? readyToRun? initialized?

Reply:
Processed